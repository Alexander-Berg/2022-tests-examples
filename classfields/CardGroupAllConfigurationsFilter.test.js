const React = require('react');
const { shallow } = require('enzyme');
const { shallowToJson } = require('enzyme-to-json');
const _ = require('lodash');

const CardGroupAllConfigurationsFilter = require('./CardGroupAllConfigurationsFilter');

const ITEMS = [ 'Автомат', 'Роботизированная', 'Вариатор', 'Механика' ];

it('должен корректно отрендерить фильтр блока "Все конфигурации"', () => {
    const tree = shallow(
        <CardGroupAllConfigurationsFilter
            items={ ITEMS }
            index={ 0 }
            onChange={ _.noop }
            placeholder="Трансмиссия"
            values={ [] }
        />,
    );
    expect(shallowToJson(tree)).toMatchSnapshot();
});
