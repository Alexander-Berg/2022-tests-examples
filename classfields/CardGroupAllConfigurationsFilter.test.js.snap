// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`должен корректно отрендерить фильтр блока "Все конфигурации" 1`] = `
<Select
  className="CardGroupAllConfigurationsFilter"
  index={0}
  maxHeight={310}
  maxMenuHeight={0}
  menuClassName="CardGroupAllConfigurationsFilter__menu"
  menuSize="m"
  mode="check"
  onChange={[Function]}
  placeholder="Трансмиссия"
  size="l"
  value={Array []}
>
  <Item
    key="Автомат"
    value="Автомат"
  >
    Автомат
  </Item>
  <Item
    key="Роботизированная"
    value="Роботизированная"
  >
    Роботизированная
  </Item>
  <Item
    key="Вариатор"
    value="Вариатор"
  >
    Вариатор
  </Item>
  <Item
    key="Механика"
    value="Механика"
  >
    Механика
  </Item>
</Select>
`;
